cmd-git.go

```diff
@@ -31,7 +31,10 @@ func gitCloneDir(url, dir string) (repo, error) {
 
 func (r *gitRepo) Checkout(rev string) error {
 	if err := gitCmd("-C", r.dir, "checkout", "master"); err != nil {
```

Comment:
git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'

how about we use this to actually find out the name of the default branch?